app/controllers/pipeline_viz_controller.rb

```diff
@@ -0,0 +1,40 @@
+class PipelineVizController < ApplicationController
+  before_action :authenticate_user!
+
+  current_power do
+    Power.new(current_user)
+  end
```

Comment:
IIRC this shouldn't be necessary, since `current_power` is already defined in the superclass `ApplicationController`.